packages/jsii-reflect/bin/jsii-tree.ts

```diff
@@ -6,22 +6,29 @@ import { TypeSystem, TypeSystemTree } from '../lib';
 
 async function main() {
   const options = yargs
-    .usage('$0 <JSII-FILE | MODULE-DIR...>', 'Prints an ASCII tree representation of a jsii type system.', args => args
+    .usage('$0 [JSII-FILE | MODULE-DIR...]', 'Prints an ASCII tree representation of a jsii type system.', args => args
       .positional('JSII-FILE', { type: 'string', desc: 'path to a .jsii file to load, all dependency .jsii files must be explicitly supplied' })
       .positional('MODULE-DIR', { type: 'string', desc: 'path to an jsii npm module directory, all jsii dependencies will be loaded transitively' }))
+    .option('closure', { type: 'string', alias: 'c', desc: 'Load dependencies of package without assuming its a JSII package itself' })
```

Comment:
Not sure I understand this feature? What is the value passed in to this switch?

Feels like you could just auto-discover this when MODULE-DIR is not a jsii module

Reply:
Well `load()` (and `loadModule()`) expects to be able to return an Assembly, and we can't return an Assembly for a non-JSII package.